atlasdb-config/src/main/java/com/palantir/atlasdb/http/FailoverFeignTarget.java

```diff
@@ -97,7 +98,11 @@ public void continueOrPropagate(RetryableException ex) {
 
         checkAndHandleFailure(ex);
         if (retryBehaviour.shouldBackoffAndTryOtherNodes()) {
-            pauseForBackOff(ex);
+            if (failoverCount.get() % servers.size() == 0) {
```

Comment:
No.
`failoverCount.get()` should be greater than 0, but I've added a check anyway.